mirrord/agent/src/outgoing.rs

```diff
@@ -148,45 +149,51 @@ impl TcpOutgoingTask {
     }
 
     /// Runs this task as long as the channels connecting it with [`TcpOutgoingApi`] are open.
+    /// This routine never fails and returns [`Result`] only due to [`WatchedTask`] constraints.
+    #[tracing::instrument(level = Level::TRACE, skip(self))]
```

Comment:
```suggestion
    #[tracing::instrument(level = Level::TRACE, skip(self), err)]
```

We're going to see the error elsewhere, but maybe add it in the tracing here just for consistency?

Reply:
There is no error, this task never fails :sweat_smile:

Reply:
Yup, saw it after the comment ...